### PR TITLE
docs(1-1262): Change addons -> integrations

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR replaces references to _addons_ in the docs with references to _integrations_.

In doing so, I have also:
- moved `/reference/addons` documents into `/reference/integrations`
- combined the previous "Addons" and "Integrations" categories into a new "Integrations" category
- added redirects from all the old addons pages to their new locations

I have updated the wording, but have not changed things such as API paths, event names, etc, because these will not change at the moment (maybe a breaking change to schedule for v6?).

### CODE REVIEW

File: website/docs/reference/integrations/addons.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- [Webhook](webhook.md) - A generic way to post messages from Unleash to third party services.
+
+## Community integrations
+
+Our wonderful community has also created the following integrations:
+
+- [Fastify feature flags plugin](https://gitlab.com/m03geek/fastify-feature-flags#unleash-provider)
+- [Quarkus](https://github.com/quarkiverse/quarkus-unleash)
+- [Vue Unleash plugin](https://github.com/crishellco/vue-unleash)
+
```
Comment (Tymek) [inline website/docs/reference/integrations/addons.md:37]: I'd like to have a section with a link to an integration request form, same way as we will add it in-app.

Comment (Tymek) [reply website/docs/reference/integrations/addons.md:37]: > Is there anything else in your workflow you would like to integrate with Unleash? Request it here.

Or something similar.

**Review by Tymek — APPROVED** [review]
Thanks. Great job 😎

File: website/docs/reference/integrations/addons.md (hunk 1 above)
Comment (thomasheartman) [reply website/docs/reference/integrations/addons.md]: Absolutely! Done in d8180fe0a1bda6c5124280aa69242ac771879c20

**Review by ivarconr — APPROVED** [review]

**Pull request merged**